### PR TITLE
AB#28770 switch from order to fsp selection

### PR DESCRIPTION
**Pull request by Piotrk39**
## Describe your changes

Removes selection of PA by order to selecting by FSP which guarantees in most cases immunity from order changes in UI and more randomised tests.

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [x] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

### CODE REVIEW

File: e2e/pages/Table/TableModule.ts [hunk 1]
```diff
@@ -469,21 +378,46 @@ class TableModule {
...
+
+      if (
+        (shouldIncludeVisa && isVisaFsp) ||
+        (!shouldIncludeVisa && !isVisaFsp)
+      ) {
+        await this.clickOnPaNumber(i);
+        return i;
+      }
+    }
+    return 400;
```
Comment (aberonni) [inline e2e/pages/Table/TableModule.ts:418]:
```suggestion
    return -1;
```

Same as in the other PR :)

**Review by aberonni — CHANGES_REQUESTED** [review]
Just a minor change if possible - and I would change all the places you return 400 to -1 (or at least, with this kinds of "search" functions)

**Comment by Piotrk39** [comment]
> Just a minor change if possible - and I would change all the places you return 400 to -1 (or at least, with this kinds of "search" functions)

Thanks a lot!

**Comment by Piotrk39** [comment]
> > Just a minor change if possible - and I would change all the places you return 400 to -1 (or at least, with this kinds of "search" functions)
> 
> Thanks a lot!

**Review by aberonni — APPROVED** [review]

**Review by aberonni — APPROVED** [review]

**Pull request merged**